### ISSUE TITLE
Let the Vercel CLI do the build

### ISSUE THREAD

**Issue by paulrberg** [amondnet/vercel-action#123]
[As per the Vercel CLI docs as of March 2022](https://vercel.com/docs/cli#project-configuration), it is no longer recommended to provide the `builds` field ourselves. Instead, the `functions` field should be used, but only when a Serverless Function is due to be deployed. For all other cases, the Vercel CLI should handle the build by itself, given that it knows what steps to run based on the project id (or at least this is what I understand by reading the docs, someone please correct me if I'm wrong).

Whatever the case, it's clear that `builds` and `routes` should not be used anymore:

<img width="1117" alt="Vercel CLI docs screenshot" src="https://user-images.githubusercontent.com/8782666/157106572-7e05af81-765b-4a77-8ad8-d80e4623530e.png">

**Comment by amondnet** [comment]
Yes, `routes` should be changed to `rewrites`. But we don't use vercel's build system, so we have to use `builds`.

**Comment by amondnet** [comment]
@paulrberg 

> That's what I wanted to say in reply to your now striken-through text 😅 I just integrated your GitHub Action and skipped the builds field and deployments are working just fine.

https://vercel.com/dashboard/usage

Because you use vercel build, you use vercel build time in addition to github actions. It also has some [limitations](https://vercel.com/docs/concepts/limits/overview#general-limits). If your build takes a long time and you do multiple builds, you will see a lot of queues. Hobby plan does only one build at a time.

If that's okay with you, you can.

**Comment by paulrberg** [comment]
>  If your build takes a long time and you do multiple builds, you will see a lot of queues

yeah I'm already starting to see the queues ..

How could I disable the Vercel builds though?
